test/modules/ModuleLib/TestAnalyzer.swagger.yaml

```diff
@@ -6,27 +6,6 @@ info:
 schemes:
 - "https"
 paths:
```

Comment:
Can you please update the `description` on top of the file that this is the reader endpoint for TestAnalyzer module please?